Source/RxComponents/LMCPowerCombiner.cc

```diff
@@ -71,8 +77,8 @@ namespace locust
 		aSignal->LongSignalTimeComplex()[sampleIndex][0] += 2.*VoltageFIRSample * sin(phi_LO);
 		aSignal->LongSignalTimeComplex()[sampleIndex][1] += 2.*VoltageFIRSample * cos(phi_LO);
 
-		if (sampleIndex%100 < 1)
-    		LDEBUG( lmclog, "Voltage " << sampleIndex << " is <" << aSignal->LongSignalTimeComplex()[sampleIndex][1] << ">" );
+		if ( (sampleIndex%100 < 1) && (fvoltageCheck==true) )
```

Comment:
Looks, good. This might be an issue if the frequency is a (or close to) multiple of 100.

Reply:
Thanks for doing this, Penny. Will It be an issue that this occurs before the low-pass filter? I usually print voltages from LMCDecimateSignalGenerator because that's the last thing that happens to them before they go to the digitizer. If that isn't an issue, then this is fine.